src/views/ipv/pyi-triage-select-device.njk

```diff
@@ -0,0 +1,43 @@
+{% extends "shared/base.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% set pageTitleName = 'pages.pyiTriageSelectDevice.title' %}
+{% set googleTagManagerPageId = "pyiTriageSelectDevice" %}
+
+{% set errorState = pageErrorState %}
+{% set errorTitle = 'pages.pyiTriageSelectDevice.content.formErrorMessage.errorSummaryTitleText' | translate %}
+{% set errorText = 'pages.pyiTriageSelectDevice.content.formErrorMessage.errorSummaryDescriptionText' | translate %}
+{% set errorHref = "#pyiTriageSelectDeviceForm" %}
+
+{% block content %}
+  <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}">{{ 'pages.pyiTriageSelectDevice.header' | translate }}</h1>
+  <p class="govuk-body">{{ 'pages.pyiTriageSelectDevice.content.paragraph1' | translate }}</p>
```

Comment:
From the image in the ticket I believe this is suppose to be a hint in the form rather than a paragraph, you may want to double check that

Reply:
Do you mean the styling should be the `govuk-hint` CSS class? I see the grey fits the DCMAW screenshot better so will go with that. I can rename the reference `.paragraph1` as well

Reply:
<img width="657" alt="Screenshot 2024-03-19 at 11 26 42" src="https://github.com/govuk-one-login/ipv-core-front/assets/144116535/c7324280-994d-44a6-96d7-420a95306b0f">

Reply:
Yeah I think thats okay for now.
This is the first scenario where we have a simple question page with a hint. We've not set an approach for this yet.
Otherwise we would put the question as a legend along with a hint inside the radiosConfig. But not sure how that will interfere with the <h1> element where we are setting `data-page="{{ googleTagManagerPageId }}`

Reply:
See similarly below what Dan mentioned*